### ISSUE TITLE
Compass won't start, Gemfile not found (Bundler::GemfileNotFound)

### ISSUE THREAD

**Issue by tjbarber** [Compass/compass#1424]
output of STDOUT when running compass: https://ghostbin.com/paste/nc6r4

I've encountered this on new installations of both Ubuntu 13.04 and Mac OS X 10.9 with installations of RVM and Ruby 2.0.0-p247.

**Comment by tjbarber** [comment]
A temporary fix for anyone else having this issue is to install rubygems-bundler version 0.9.2 and make sure any newer versions are uninstalled. It would probably work with a version a bit newer (found 0.9.2 being used here: http://stackoverflow.com/questions/10610254/cant-install-compass-via-rvm) but something in 1.4.1 breaks compass badly.

**Comment by Danielpk** [comment]
I have same problem. Installing rubygems-bundler 0.9.2 did not work for me.

**Comment by tjbarber** [comment]
It's working for me, so try this: switch to your global gem set

`rvm ruby-2.0.0-p247@global`

If rubygems-bundler is installed in global, uninstall it and then switch back to the local gemset. Here's my gem list on my local install: http://d.pr/n/TBiQ

**Comment by overneath42** [comment]
Had the same problem, and removing rubygems-bundler 1.4.1 did solve it. Even if it's only a temporary fix, a fix is still a fix. So thanks!

**Comment by x3ro** [comment]
Same problem here. Taking a look at the backtrace, the "faulty" line(s) are these, from `bin/compass`:

```
if defined?(Bundler)
    require 'bundler/shared_helpers'
    Bundler.require :assets if Bundler::SharedHelpers.in_bundle?
end
```

`Bundler::SharedHelpers.in_bundle?` apparently returnes a false positive. Executing `in_bundle?` in irb in the working directory where I'm trying to run compass yields `nil`, which seems okay. I'm assuming that, instead of looking at the working directory where I'm running compass, it somehow looks for a bundle relative to `bin/compass`, but then loads bundler in my current working directory (just guessing though).

tl;dr: Removing the `Bundler.require` line makes compass run flawlessly :smile: (Thanks for making compass!)
